src/Modulo2.jl

```diff
@@ -271,6 +272,21 @@ function copyto!(a::ZZ2Array, b::ZZ2Array)
     end
 end
 
+"""
+    copyto!(a::ZZ2Vector, num::Integer) 
+
+Copy the bit representation of number `num` into ZZ2Vector `a`
+"""
+function copyto!(a::ZZ2Vector, num::T) where T <: Integer
```

Comment:
I'm leaning towards adding a function like this, but probably with a different name. Currently I'm thinking about something like `zz2vector!`. The non-mutating variant (which should also be there) would then be called `zz2vector` (in analogy to `bitstring`).

Reply:
Agreed!